idd/Energy+.idd.in

```diff
@@ -48608,6 +48608,61 @@ ZoneHVAC:AirDistributionUnit,
        \type object-list
        \object-list DesignSpecificationAirTerminalSizingName
 
+ZoneHVAC:ExhaustControl,
+       \memo Defines a controlled exhaust flow from a zone which feeds into 
+       \memo an AirloopHVAC:Exhaust system inlet.
+  A1 , \field Name
+       \required-field
+  A2 , \field Availability Schedule Name
+       \note Availability schedule name for this exhaust system. Schedule value > 0 means it is available.
+       \note If this field is blank, the exhaust system is always available. If the attached 
+       \note AirloopHVAC:ExhaustSystem is off, then the flow will be zero.
+       \type object-list
+       \object-list ScheduleNames
+  A3 , \field Zone Name
+       \note Zone the exhaust inlet node is part of
+       \type object-list
+       \object-list ZoneNames
+  A4 , \field Inlet Node Name
+       \note Inlet node name for the exhaust
```

Comment:
...exhaust. Must be a zone exhaust node.

Reply:
Done.